docs/user-roles.rst

```diff
@@ -44,3 +57,63 @@ also perform every available operation inside that project.
     organization.
 
 A *project member* has read-only access to the project.
+
+
+.. _user-ops:
+
+User operations
+===============
+
+This section describes issues relating to user operations that derive from the
+architecture of CrateDB and CrateDB Cloud. They are here appended to ensure the
+CrateDB Cloud customer is aware of the constraints they impose.
+
+
+.. _system-user:
+
+``System`` user
+---------------
+
+In CrateDB Cloud, the backend uses a user called ``system`` in order to perform
+Cloud cluster upgrades, backups and scaling functions, among other things.
+
+.. WARNING::
+
+    The user ``system`` is essential for CrateDB Cloud to function as intended.
+    While it is not normally accessible through the CrateDB Console or the
+    Croud CLI, it can be accessed through the CrateDB Admin UI or any other SQL
+    client. It is important not to edit or delete this user in any way.
+    Otherwise, the functioning of Cloud clusters may be compromised.
+
+
+.. _crate-user:
+
+``Crate`` user
+--------------
+
+In CrateDB, when you create a cluster node (through whatever method), a
+``crate`` user is automatically generated. ``crate`` is a superuser and can
+perform all possible operations. Next to the ``crate`` user there is the
+regular database user, created as part of the CrateDB Cloud cluster
+deployment wizard when deploying a cluster through `AWS`_ or `Azure`_.
+
+Because ``crate`` is a superuser, there are certain operations that the *crate*
+user can perform which are not possible for the regular database user. As of
+CrateDB 4.2.1, the list of such operations is as follows:
```

Comment:
I think we can also distinguish hierarchically between the crate user and regular users., especially because otherwise it looks very much as if the database user would also be a superuser.

```suggestion
perform all possible operations.


.. _regular-user:

Regular user
------------

Next to the ``crate`` user there is the
regular database user, created as part of the CrateDB Cloud cluster
deployment wizard when deploying a cluster through `AWS`_ or `Azure`_.

Because the regular user is as a database user with `AL`_ privileges, there are certain operations that this user cannot do. As of
CrateDB 4.2.1, the list of such operations is as follows:
```

Reply:
In general, the distinction between Cloud users and CrateDB users is much clearer now.